src/hooks/useSelector.js

```diff
@@ -97,12 +97,16 @@ export function createSelectorHook(context = ReactReduxContext) {
     }
     const { store, subscription: contextSub } = useReduxContext()
 
-    return useSelectorWithStoreAndSubscription(
+    const selectedState = useSelectorWithStoreAndSubscription(
       selector,
       equalityFn,
       store,
       contextSub
     )
+
+    useDebugValue(selectedState)
```

Comment:
It should be at the root of `useSelector` to make Redux DevTools show the value right besides the hook name